src/App.tsx

```diff
@@ -10,9 +10,7 @@ export const App = () => {
 
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Outlet />
```

Comment:
You should properly style NavLinks in the Navbar:
![image](https://github.com/user-attachments/assets/5d9e287d-1994-4bbb-98e1-c7dcf0914f84)